apps/src/studio/studio.js

```diff
@@ -3013,7 +3018,13 @@ Studio.displaySprite = function(i, isWalking) {
     spriteWalkIcon.setAttribute('visibility', 'hidden');
 
     xOffset = sprite.drawWidth * spriteFrameNumber(i);
-    yOffset = 0;
+
+    // For new versions of sprites (iceage/gumball), a yoffset is required for emotions
```

Comment:
actually maybe rather than just referring to yOffset, better to explain that there are additional rows of emotions in the spritesheet.